### ISSUE TITLE
Compatible with Super-Splat (not really an issue)

### ISSUE THREAD

**Issue by AtlasRedux** [francescofugazzi/3dgsconverter#5]
You should add that it works with Super-Splat generated PLYs as well.
Super-Splat is incredibly much more convenient and faster to do rough cleaning than CloudCompare, so this is an absolute game changer for myself.

https://github.com/playcanvas/super-splat/

**Comment by francescofugazzi** [comment]
because I have little time lately could you please send me a sample ply?

**Comment by entracte-arlo** [comment]
If you use SuperSplat, spherical harmonic data is not saved on export at this time

https://github.com/playcanvas/super-splat/#current-limitations

**Comment by AtlasRedux** [comment]
Wrong. They implemented spherical harmonic data a couple of days ago (after I nagged them xD).
Go to the live version. There is an option to check the button for SH (spherical harmonics) now, both for import and export.

**Comment by PLAN8VR** [comment]
I cant find the SH option in the live version, has it been implemented yet?

**Comment by PLAN8VR** [comment]
In fact, I just downloaded and installed locally the latest version 1.16.0 (although it loads as 0.16.0), and this doesn't have the SH option in it either as far as I can see
